### ISSUE TITLE
High poly custom hairs don't work

### ISSUE THREAD

**Issue by redandvidya** [lah7/sims-2-wine-patches#8]
Deets that you should know first:
OS: Arch x64
GPU: NVIDIA GT 1030
CPU: i5-3470
I'm using the older "tkg-sims2.3.21" version that is included with lutris, and not the newer 4.0 version.

High poly custom content hairs, do not work in the game and simply flash red every time you try and use them.

Video link: https://streamable.com/dd9an
You can try it out for yourself by using[ this mod](https://keoni-chan.tumblr.com/post/164248634827/i-made-a-thing-3-my-game-generated-a-npc-with) that replaces a hair from Kitchen and Bath Stuff. 

If anyone can know how this is fixable it would be very appreciated.

**Comment by lah7** [comment]
Try D9VK, I've just been informed of its progress and updated the README. In theory, this issue might be "fixed" under D9VK as it will use Vulkan _(and the hardware?)_ for vertex shaders, which normally causes those flashing red textures.

* Instructions: https://github.com/lah7/sims-2-wine-patches/blob/master/README-D9VK.md

I believe @tannisroot maintains the Lutris installer, so I'm not sure if new installs under Lutris will be using D9VK. However, you can still add D9VK to your Wine prefix, but will possibly need Wine 3.0 or newer.

**Comment by misyltoad** [comment]
Yes, I implement ProcessVertices using a geometry shader (that emits no vertices) and buffer writes which allows this to work.
